hack/testing-olm/utils

```diff
@@ -83,15 +83,38 @@ gather_logging_resources() {
   set +e
   local LOGGING_NS=$1
   local outdir=${2:-$ARTIFACT_DIR}
+  local runtime=${3:-$(date +%s)}
+  outdir=$outdir/$runtime
```

Comment:
why are we setting `outdir` when we have it set above as well?

Reply:
i don't know, i copied this from the hack/testing/utils script:
https://github.com/openshift/cluster-logging-operator/blob/master/hack/testing/utils#L83-L85

seems like it's trying to build up a path.